pythonforandroid/recipes/genericndkbuild/__init__.py

```diff
@@ -7,7 +7,7 @@ class GenericNDKBuildRecipe(BootstrapNDKRecipe):
     version = None
     url = None
 
-    depends = [('python2', 'python3', 'python3crystax')]
+    depends = [('python2', 'python3')]
```

Comment:
As I understood python2 and 3 are implicitly in the `depends` list aslso if we remove then correct?

Reply:
mmm...I don't think so, those are only implicit in any recipe that inherits of `PythonRecipe`, but not for `BootstrapNDKRecipe`...or maybe I miss something?